src/AppBundle/Entity/User.php

```diff
@@ -36,7 +36,25 @@ class User implements UserInterface
 	private $password;
 
 	/**
-	 * @Assert\NotBlank()
```

Comment:
Nejsem si jistý, jak správně řešit tuto validaci když pro registraci chci aby to nebylo null ale pro editaci adresy to null naopak chci..